src/triage.database/TriageDb.cs

```diff
@@ -81,8 +86,9 @@ public static void Init(string connStr)
                 await context.Entry<Dump>(dump).ReloadAsync();
 
                 dump.LoadTriageData(triageData);
-            
+
                 await context.SaveChangesAsync();
+               
```

Comment:
I will leave this for the formatter to clean up. If that doesn't work, then I will do it myself.I'm not sure why all the weird little spaces.